atom/browser/api/atom_api_window.cc

```diff
@@ -338,16 +338,30 @@ bool Window::IsFullscreen() {
   return window_->IsFullscreen();
 }
 
-void Window::SetBounds(const gfx::Rect& bounds) {
-  window_->SetBounds(bounds);
+void Window::SetBounds(mate::Arguments* args) {
```

Comment:
@zcbenz okay, but then a question comes: should I check for type correctness, like this:
```
if (args->Length() == 1 && !args->GetNext(&animate)) {
  args->ThrowError();
  return;
}
```
In your example, if I pass, say, a string for `animate` parameter, it will be just ignored and handled like `false`.